src/components/atoms/main.tsx

```diff
@@ -14,5 +14,5 @@ export const Main = tw.main.attrs({ style: { maxWidth: MAX_SITE_WIDTH } })`
   mx-auto
   tablet-md:pt-32
   tablet-md:px-0
-  motion-safe:animate-fade-in
```

Comment:
would you mind finding all occurrences of `motion-safe:animate-fade-in` and also remove the tailwind config for it?